main.nf

```diff
@@ -312,28 +325,26 @@ process RealignerTargetCreator {
   """
 }
 
-verbose ? intervals = intervals.view {"Intervals to phase: $it"} : ''
+if (verbose) intervals = intervals.view {"Intervals to phase: $it"}
 
 bamsAndIntervals = duplicatesRealign
   .phase(intervals)
-  .map{duplicatesRealign, intervals ->
+  .map{dr, i ->
     tuple(
-      duplicatesRealign[0],
-      duplicatesRealign[1],
-      duplicatesRealign[2],
-      duplicatesRealign[3],
-      duplicatesRealign[4],
-      intervals[2]
+      dr[0],
+      dr[1],
+      dr[2],
+      i[2]
```

Comment:
Ah, good, thanks! I don’t run the full tests locally and rely on Travis instead ;-)